### DISCUSSION TITLE
Git Module / Extension Uninstalled

### DISCUSSION THREAD

**Discussion by chris2fr** [requarks/wiki#5190] [Help / Questions]
Hi,

I somehow installed my Wiki.JS with Pandoc, Pupeteer and Sharp extensions, but impossible to get the Git extension working !

What are the dependencies and the process to install the git extension ?

Many thanks,

Chris

![image](https://user-images.githubusercontent.com/104329/163566805-e3e37434-746a-41d7-a62a-6073db32c07f.png)

npm i puppeteer sharp pandoc --force

**Comment by maxreig** [comment]
Hi,
You have to have the GIT binary installed, in my case I have rhel8, and I installed it like this.
 dnf git install.
 How did you install the other 3?
 Luck.
